pkgs/development/python-modules/pyunormalize/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchPypi,
+}:
+
+buildPythonPackage rec {
+  pname = "pyunormalize";
+  version = "15.1.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-z0qHRRoPHLdpEaqX9DL0V54fVkovDITOSIxzpzkBtsE=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [ "pyunormalize" ];
+
```

Comment:
Please enable tests.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
I thought the tests were enabled by default. I was asked to remove "doCheck = true;" in the previous comments

Reply:
https://github.com/NixOS/nixpkgs/pull/337542#discussion_r1742560063

Reply:
Did you read the documentation I showed you?
It says that `buildPythonPackage` sets `doCheck = true` by default, and the package needs its own checkPhase. In most cases, just add pytestCheckHook to nativeCheckInputs.

Reply:
Got it. Done.